doc/nrf/ug_nrf5340.rst

```diff
@@ -106,165 +85,246 @@ Zephyr includes the `OpenAMP`_ library, which provides a complete solution for e
 
-`b)` This option is used by default for Thread and Zigbee samples in the |NCS| since in this case :option:`CONFIG_NRF_802154_SER_HOST` defaults to ``y``.
 
-`c)` This option is used for Thread and Zigbee samples in the |NCS| when the Bluetooth LE is additionally enabled by the user.
+Configuration of the child image
+================================
 
-SES is unable to automatically program the network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` to the network core when it is added as a child image.
-To program the network sample to the network core, see `Programming the network sample from SES`_.
+When a network sample is built automatically as a child image in a multi-image build, you can define the relevant Kconfig options (if required) in a :file:`.conf` file.
+Name the file *network_sample*\ .conf, where *network_sample* is the name of the child image (for example, ``hci_rpmsg.conf``).
+Place the file in a :file:`child_image` subfolder of the application sample directory.
+This allows alignment of the configurations of both images.
+
+For example, see the :ref:`ble_throughput` child image configuration in :file:`nrf/samples/bluetooth/throughput/child_image/hci_rpmsg.conf`.
+
+.. _ug_nrf5340_building:
+
+Building and programming a sample
+*********************************
+
+Depending on the sample, you must program only the application core (for example, when using NFC samples) or both the network and the application core.
+
+The steps differ depending on whether you work with |SES| or on the command line and whether you are doing a single or multi-image build.
+
+Using SEGGER Embedded Studio
+============================
+
+To build and program separate images with |SES|, follow the general instructions for :ref:`gs_programming_ses`.
+To program a multi-image HEX file, you must add the project files for the network core after building, as described in :ref:`ug_nrf5340_ses_multi_image`.
 
-A dedicated network sample can be built and programmed by following the instructions in :ref:`gs_programming_ses`.
-Make sure to use ``nrf5340dk_nrf5340_cpunet`` as the build target when building the network sample, and ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuappns`` when building the application sample.
+Separate images
+---------------
+
+To build and program only the application core, follow the instructions in :ref:`gs_programming_ses` and use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuappns`` as build target.
+
+To build and program a dedicated network sample, follow the instructions in :ref:`gs_programming_ses` and use ``nrf5340dk_nrf5340_cpunet`` as the build target.
 
-Programming the network sample from SES
-=======================================
+.. _ug_nrf5340_ses_multi_image:
 
-Follow the instructions in :ref:`gs_programming_ses`, use ``nrf5340dk_nrf5340_cpuapp`` or ``nrf5340dk_nrf5340_cpuappns`` as the build target, and open the Bluetooth Low Energy sample to work with.
+Multi-image build
+-----------------
```

Comment:
Should we mention that it is possible to skip the build of an automatically included child image? Or to point to a hex file in its place?

See last section of https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_multi_image.html#default-configuration

Reply:
I don't think we need this here - this guide should focus on the most common use cases, and when users get to the point where they need to skip/replace, they probably have read through the multi-image user guide already.